app/controllers/dashboard/associated_users_controller.rb

```diff
@@ -88,12 +89,13 @@ def update
       @current_user_updated = params[:project_role][:identity_id].to_i == @user.id
       
       if @current_user_updated
```

Comment:
We need to track updating the current user because their permissions might change